_remake/main.js

```diff
@@ -114,27 +123,75 @@ app.use(expressSession({
   }
 }));
 
-
 // flash message middleware
 app.use(flash());
 
+// if app is multi tenant
+// open MySQL connection
+// and use bodyParser (needed for post requests in lib/init-service-routes.js)
+if (RemakeStore.isMultiTenant()) {
+  global.config = {
```

Comment:
I see where `global` is coming from now. I didn't know this was built in to Node (although I kind of remember now). I should probably have used this instead of `RemakeStore`...